### DISCUSSION TITLE
Can't request upload files after update to 9.3.2

### DISCUSSION THREAD

**Discussion by goalbased** [vercel/next.js#11676] [Help]
I had a function which will upload images from front-end and at the back-end will save the images to the static folder. After I updated next package from 9.1.2 to 9.3.2 front-end request the image file util I restart the next server

**Comment by timneutkens** [comment]
This is expected as it mitigates the security issue of https://github.com/zeit/next.js/releases/tag/v9.3.2

**Reply by goalbased** [reply]
**This does not affect files outside of the dist directory (.next).**
But my upload files are under the static folder, is it also expected?
Now, I downgrade to 9.1.2. I can't restart Next server every time when user upload files

**Reply by timneutkens** [reply]
> This does not affect files outside of the dist directory (.next).
> But my upload files are under the static folder, is it also expected?

The mitigation for `.next` was applied everywhere that we serve files to ensure it doesn't happen again.
